### DISCUSSION TITLE
Is it possible to get a video on the installation process?

### DISCUSSION THREAD

**Discussion by problemchild09** [ImoutoChan/ImoutoRebirth#1] [General]
I have basically no coding knowledge, but this programme looks like it'd be really useful. could we get a video on how to install it?

**Comment by ImoutoChan** [comment]
Hi! 

As you can probably see this software still isn't very user-friendly for new users. I've made several steps in this direction, but it remains challenging to use it flawlessly.

Regarding the video, I've actually already recorded a guide on how to install and configure + small feature demonstration, but I haven't had time to edit or upload it. I'll try to do it this weekend and post a link here when it's ready.

**Reply by ImoutoChan** [reply]
Welp, better late than never, I’ve finally done with installation and configuration videos:

* guide with installation and initial configuration: https://youtu.be/rEg0WMQgmoE
* guide with first launch, collection configuration, demo some features, Imouto Extensions demo: https://youtu.be/7Gnbc3296GU

I'm also planning to create some kind of wizard in the app to simplify the process of creating collections, but I have no ETA for it
